dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/Parameters.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.data.mongodb.core.mapping.Field;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@ConfigurationProperties
+public class Parameters {
+
+    public static final String PROCESSOR = "processor";
```

Comment:
I put them in different places because they have completely different purposes*[1] and can change independently. I would keep them as they are.

I ended up asking for the processor to avoid adding more ways to express the same, but I could have added a parameter `bool submittedToEVA`, and check for true in the bean injection. So this processor is the name of the parameter in the application.properties.

*[1] More different than ImportVariantsStepConfiguration.IMPORT_VARIANTS_STEP_BEAN and IMPORT_VARIANTS_STEP which are in the same class because they both refer to the same step, from the bean side and the jobLauncher side.